docs/tables/aws_kinesis_video_stream.md

```diff
@@ -0,0 +1,68 @@
+# Table: aws_kinesis_video_stream
+
+Amazon Kinesis Video Streams makes it easy to securely stream video from connected devices to AWS for analytics, machine learning (ML), playback, and other processing.
+
+## Examples
+
+### List all the video streams in your account
+
+```sql
+select
+  stream_name,
+  stream_arn,
+  status,
+  creation_time,
+  region
+from
+  aws_kinesis_video_stream;
+```
+
+
+### List all the video streams that are not in Active state
+
+```sql
+select
+  stream_name,
+  stream_arn,
+  status,
+  creation_time,
+  region
+from
+  aws_kinesis_video_stream
+where
+  status != 'ACTIVE';
+```
+
+
+### List all the video streams which are not encrypted using CMK
+
+```sql
+select
+  stream_name,
+  stream_arn,
+  status,
+  kms_key_id,
+  creation_time,
+  region
+from
+  aws_kinesis_video_stream
+where
+  split_part(kms_key_id, '/', 3) = 'kinesisvideo';
```

Comment:
Can we add a new column to store kms_key_name? instead of split the ID here to check the key_name?
For example: `alias/aws/kinesisvideo` for default